source/user-manual/capabilities/file-integrity/fim-fields-rule-mapping.rst

```diff
@@ -0,0 +1,145 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+Fim fields rule mapping
+=======================
+
+This guide aims to help to understand how FIM fields can be mapped into rules.
+
+
+FIM - Alerts : fields correspondence
+------------------------------------
+
+The following table establishes a correspondence between the syscheck fields and their homologs in the alert generation.
```

Comment:
```suggestion
The following table establishes a correspondence between the decoded FIM fields and their counterpart in rules.
```

Reply:
Done in [commit](https://github.com/wazuh/wazuh-documentation/pull/3913/commits/d5ce0fcadd5abf8bbb40c82c8ccbba1b333c2572)